R/deprecated.R

```diff
@@ -0,0 +1,20 @@
+#' Check Package Dependencies
+#'
+#' Collects R dependencies and checks them against OSS Index.
+#' Returns a tibble of results.
+#'
+#' This function is deprecated. See
+#' @details By default, packages listed in \code{installed.packages()} are scanned by sonatype.
+#' However, you can pass your own data frame of packages. This data frame should have two columns,
+#' \code{version} and \code{package}.
+#' @param pkgs Default \code{NULL}. See details for further information.
+#' @param verbose Default \code{TRUE}.
+#' @return A tibble/data.frame.
+#' @export
+audit_deps = function(pkgs = NULL, verbose = TRUE) {
```

Comment:
Depreciate the previous function. Remove in future

Reply:
Love it.